wgpu-core/src/device/global.rs

```diff
@@ -1197,7 +1197,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                     }
                     #[cfg(feature = "spirv")]
                     pipeline::ShaderModuleSource::SpirV(ref code, _) => {
-                        trace.make_binary("spirv", code.as_bytes())
+                        trace.make_binary("spirv", cast_slice::<u32, u8>(code.borrow()))
```

Comment:
Nit:

```suggestion
                        trace.make_binary("spirv", bytemuck::cast_slice::<u32, u8>(&code))
```

Should work and remove the need for any imports at all.

Reply:
Thanks for catching that! I pushed the change in a different commit to also remove the imports as suggested.